#!/usr/bin/python3
# -*- coding: utf-8 -*-

# 部分数据需要使用DDT，部分不需要
# 第1组数据
# 登录成功,"18684720553"/"python"

# 第2组数据
# 密码为空，18684720553/空,"请输入密码"
# 账号名为空，""/"python" "请输入手机号"
# 账号格式错误，10位"1850000000"或者12位"185000000000"/"python" ，"请输入正确的手机号"

# 第3组数据
# 密码错误，"18684720553"/"111111"，"帐号或密码错误!"
# 账号未注册，"18500000000"/"python"，"此账号没有经过授权，请联系管理员!"

# 登录成功数据
success_data = {"user": "18684720553", "pwd": "python"}
# 异常数据1---密码为空、账号名为空、手机号格式不正确
wrong_data_1 = [
    {"user": "18684720553", "pwd": "", "check": "请输入密码"},
    {"user": "", "pwd": "python", "check": "请输入手机号"},
    {"user": "1868472055", "pwd": "python", "check": "请输入正确的手机号"},
    {"user": "186847205533", "pwd": "python", "check": "请输入正确的手机号"}
]
# 异常数据2---密码错误、账号未注册
wrong_data_2 = [
    {"user": "18684720553", "pwd": "111111", "check": "帐号或密码错误!"},
    {"user": "18500000000", "pwd": "python", "check": "此账号没有经过授权，请联系管理员!"}
]

